home/jobs/OMERO-web/config.xml

```diff
@@ -68,24 +69,56 @@ unzip $ZIP
 rm -f $ZIP
 mv $WORKSPACE/$DIST $OMERO_DIST
 
-virtualenv $WORKSPACE/omeropy-virtualenv --system-site-packages
-source $WORKSPACE/omeropy-virtualenv/bin/activate
+source $HOME/omero-virtualenv/bin/activate
+
+source $OMERO_INSTALL/settings.env
+
+
+pip install --upgrade --no-deps --force-reinstall -r $OMERO_DIST/share/web/requirements-py27-nginx.txt
+
 
-cd $OMERO_INSTALL
-source settings.env
+#start-config
+$OMERO_DIST/bin/omero config set omero.web.application_server &apos;wsgi-tcp&apos;
+$OMERO_DIST/bin/omero config set omero.web.application_server.max_requests 0
 
-# From step04
-$OMERO_DIST/bin/omero web config nginx --http &quot;$OMERO_WEB_PORT&quot; &gt; $OMERO_DIST/nginx.conf.tmp
+$OMERO_DIST/bin/omero config set omero.web.prefix &apos;/web&apos;
+$OMERO_DIST/bin/omero config set omero.web.static_url &apos;/web/static/&apos;
+$OMERO_DIST/bin/omero config set omero.web.static_root &apos;/home/omero/nginx/web&apos;
 
-sudo -u root bash step05_centos7_nginx.sh
-sudo -u root bash step06_centos7_daemon.sh
+$OMERO_DIST/bin/omero config set omero.web.session_engine &apos;django.contrib.sessions.backends.cache&apos;
+$OMERO_DIST/bin/omero config set omero.web.caches &apos;{&quot;default&quot;: {&quot;BACKEND&quot;: &quot;redis_cache.RedisCache&quot;,&quot;LOCATION&quot;: &quot;192.168.99.100:6379&quot;}}&apos;
+
+$OMERO_DIST/bin/omero config set omero.web.application_server.host 192.168.99.100
```

Comment:
:+1: Then let's discuss best paths forward tomorrow. My PR isn't working anyway so happy to merge, but you can also take over the PR if you want if that'll simplify the process.